main.py

```diff
@@ -21,6 +21,9 @@
 openai_api_key = os.environ.get("OPENAI_API_KEY")
 primo_api_key = os.environ.get("PRIMO_API_KEY")
 primo_api_host = os.environ.get("PRIMO_API_HOST")
+primo_api_limit = os.environ.get("PRIMO_API_LIMIT", 100)
+# Due to token limits when using context injection, we must limit the amount of primo results we send to the llm. This limit should be different for different llm models depending on their token capacity.
+max_results_to_llm = int(os.environ.get("MAX_RESULTS_TO_LLM"), 5)
```

Comment:
Set the base of int to 10 and the default value of get environment variable to 5 
int(os.environ.get("MAX_RESULTS_TO_LLM", 5), 10)